src/Lykke.Logs/Obsolete/LogEntity.cs

```diff
@@ -1,22 +1,36 @@
-﻿using System;
+using System;
+using JetBrains.Annotations;
+using Lykke.Common;
 using Microsoft.WindowsAzure.Storage.Table;
 
 namespace Lykke.Logs
 {
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
New type is internal, so we can't replace it with particular new type